pkgs/applications/window-managers/waybox/default.nix

```diff
@@ -41,5 +41,6 @@ stdenv.mkDerivation rec {
     license = licenses.mit;
     maintainers = with maintainers; [ AndersonTorres ];
     platforms = platforms.linux;
+    broken = true;
```

Comment:
We should also notify the maintainers if we mark something as broken.
cc @AndersonTorres (already in this thread but to make it explicit)

I'd assume that `waybox` is still in use so we should either override `wlroots` (0.12) or add a TODO comment if this is only temporary and will be dropped before this PR gets merged.

Reply:
There doesn't seem to be any upstream development in the past half year or so, so I marked it as broken for now. I'll add a note, and I guess we'll see what the status is when the final version of wlroots 0.13 is released.

Reply:
Ok, that makes sense then :) Thanks!